package.json

```diff
@@ -34,8 +34,13 @@
     "@nuxtjs/dotenv": "^1.4.0",
     "@nuxtjs/pwa": "^3.0.0-beta.20",
     "@nuxtjs/svg": "^0.1.7",
+    "child_process": "^1.0.2",
+    "fs": "0.0.1-security",
```

Comment:
How are these used?

Reply:
`fs` and `child_process` are dependencies that are needed to use jsdom which is being implemented in the shared/SideMenu.js file